MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Configuration/EffectConfiguror.java

```diff
@@ -32,21 +33,23 @@ private static AscensionEffectBuilderImpl buildAscensionEffects(Map effect) {
         AscensionEffectBuilderImpl builder = null;
 
         switch ((String) effect.get("name")) {
-            case "BloodyAscensionEffect":
-                builder = new AscensionEffectBuilderImpl.BloodyAscensionEffectBuilder();
+            case "PotionAscensionEffect":
+                builder = new AscensionEffectBuilderImpl.PotionAscensionEffectBuilder();
+                ((AscensionEffectBuilderImpl.PotionAscensionEffectBuilder) builder)
+                        .setEffects((List<String>) effect.getOrDefault("effects", null));
                 break;
-            case "DizzyAscensionEffect":
-                builder = new AscensionEffectBuilderImpl.DizzyAscensionEffectBuilder();
-                break;
-            case "DamagingAscensionEffect":
-                builder = new AscensionEffectBuilderImpl.DamagingAscensionEffectBuilder();
+            case "ParticleAscensionEffect":
```

Comment:
I still dont like this but for now Ill let it slide. Fix it later @Miirpat-Gaalsien